apps/sentry-client-desktop/src/features/home/SentryWallet.tsx

```diff
@@ -414,7 +414,7 @@ export function SentryWallet() {
 													}
 													setSelectedValue={setSelectedWallet}
 													getDropdownItems={getDropdownItems}
-													extraClasses={{dropdown: "!w-[330px] !h-[44px]"}}
+													extraClasses={{dropdown: "!w-[330px] !h-[44px]", dropdownOptions: "w-[538px]"}}
 													dropdownOptionsCount={data.whitelistedWallets.length}
```

Comment:
I have not checked the appearance of the dropdown in the desktop app, but I'd assume these widths should be adjusted down to match the existing widths, but I did not want to change without a review.

Reply:
@XDapps is width `538px` correct here, seems to be `330px` ?

Reply:
Since these were previously using the hardcoded dropdown, I wanted to keep the values that were currently being used in the component before it was changed.

I can most certainly adjust to match if you prefer.

Reply:
@XDapps Im not sure about this, its important that we check if anything changes with our updates. I don't require to set 330px, rather I wwant to confirm that we have the correct value in there.

Reply:
This one seems to be wrong


![image](https://github.com/user-attachments/assets/296e6de2-9323-4783-8725-7effedb2f448)

Reply:
I changed this back to the original value. I tried to test, but can't seem to get the dropdown to appear.
![Load](https://github.com/user-attachments/assets/5616a8a1-571c-4d80-ab19-418c935933be)

Reply:
You will have to approve the wallet from your app `0x933...` with a wallet that has keys. Le me know if you need a private key for a wallet that owns keys.

For approval you would have to manually call setApprovalForOperator on the sepolia Referee https://sepolia.arbiscan.io/address/0xF84D76755a68bE9DFdab9a0b6d934896Ceab957b#writeProxyContract

Let me know if that works